src/core/server.ts

```diff
@@ -45,6 +44,8 @@ export interface ServerOptions<T> {
   warthogImportPath?: string;
   introspection?: boolean; // DEPRECATED
   bodyParserConfig?: OptionsJson;
+  onBeforeGraphqlHook?: (app: express.Application) => void;
```

Comment:
Should we call this `onBeforeGraphQLMiddleware`?

Reply:
no.  this is a general purpose hook ...